bitnami/jenkins/values.yaml

```diff
@@ -111,6 +111,23 @@ nodeSelector: {}
 ##
 tolerations: {}
 
+## Jenkins pods' Security Context
+## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-pod
+##
+podSecurityContext:
+  fsGroup: 1001
+  runAsUser: 1001
```

Comment:
I would move this to containerSecurityContext (only the runAsUser part so it is compatible with istio)